java/ddlogapi/DDlogAPI.java

```diff
@@ -115,11 +115,11 @@
      *                  many times, on potentially different threads, when the "commit"
      *                  API function is called.
      */
-    public DDlogAPI(int workers, Consumer<DDlogCommand> callback, boolean storeData) {
+    public DDlogAPI(int workers, Consumer<DDlogCommand> callback, boolean storeInputs, boolean storeOutputs) {
```

Comment:
You may want to provide another version of this constructor with suitable default values for these arguments.

Reply:
It looks like "true, false" is the most common combination.